cmd/gh-classroom/pull/student-repos/student-repos.go

```diff
@@ -71,11 +71,19 @@ func NewCmdStudentReposPull(f *cmdutil.Factory) *cobra.Command {
 				log.Fatal(err)
 			}
 
-			entries, err := os.ReadDir(fullPath)
+			all_entries, err := os.ReadDir(fullPath)
 			if err != nil {
 				log.Fatal(err)
 			}
 
+			//filter entries to only directories
+			var entries []os.DirEntry
+			for _, r := range all_entries {
+				if r.IsDir() {
+					entries = append(entries, r)
+				}
+			}
+
 			for _, r := range entries {
 				clonePath := filepath.Join(fullPath, r.Name())
```

Comment:
Would it make sense to add a conditional here instead of filtering the list? That way we only have to loop the one time.

Reply:
That's a good idea. Like at line 88 adding a guard clause. I'll try it out.